package.json

```
@@ -3,14 +3,35 @@
   "version": "17.0.0",
   "private": true,
   "description": "Проект «Большое путешествие» от HTML Academy",
+  "browserslist": [
+    "last 2 versions",
+    "not dead",
+    "not ie <= 11",
+    "not op_mini all",
+    "not < 0.25%"
+  ],
   "scripts": {
-    "lint": "eslint src/"
+    "lint": "eslint src/",
+    "build": "webpack --mode production",
+    "start": "webpack serve --mode development --open"
   },
   "devDependencies": {
+    "@babel/core": "^7.17.10",
```

Лучше фиксировать зависимости, флаг exact или сокращенно E. То есть при установке условно `npm i react -E` подправь во всех местах
```suggestion
    "@babel/core": "7.17.10",
```